katharsis-meta/src/main/java/io/katharsis/meta/model/resource/MetaResourceAction.java

```diff
@@ -0,0 +1,23 @@
+package io.katharsis.meta.model.resource;
+
+import io.katharsis.meta.model.MetaElement;
+import io.katharsis.resource.annotations.JsonApiResource;
+
+@JsonApiResource(type = "meta/resourceAction")
+public class MetaResourceAction extends MetaElement {
```

Comment:
https://github.com/katharsis-project/katharsis-framework/issues/24

=> there is an example in ScheduleRepositoryImpl/ScheduleRepository in katharsis-client.

potentially a bit more support in this area would be helpful in the future but off topic here.